src/packages/atomicgraphnets.jl

```diff
@@ -0,0 +1,64 @@
+using Pkg
+using Pkg.Artifacts
+using Serialization
+
+flux_versions = ["0.11", "0.12"]
+env_dirs = Dict(k=>joinpath(@__DIR__, "AtomicGraphNets_deps", "v$k") for k in flux_versions)
+
+function agn_setup(flux_version, batchsize, num_conv, hidden_layer_width, num_hidden_layers)
+    # sanity checks
+    @assert flux_version in flux_versions
+    @assert batchsize <= 100
+
+    cd(env_dirs[flux_version])
+    Pkg.activate(".")
+    
+    @eval using AtomicGraphNets
+
+    # load up the (pre-shuffled) data
+    #artpaths = Dict("0.11"=>artifact"v011_data", "0.12"=>artifact"v012_data")
+    artpaths = Dict(k=>joinpath(@__DIR__, "..", "..", "..", "AGN_FluxBench_data", k) for k in flux_versions)
```

Comment:
The commented out line is the expected way artifacts should be referenced.

Reply:
yeah, sorry I didn't mean to push this, that change was just to try to continue playing around locally haha